test/integration/targets/wait_for/tasks/main.yml

```
@@ -165,10 +165,14 @@
       - waitfor is not changed
       - "waitfor.port == {{ http_port }}"
 
-- name: install psutil using pip (non-Linux only)
+- name: install psutil using pip (non-Linux and older FreeBSD only)
   pip:
     name: psutil==5.8.0
-  when: ansible_system != 'Linux'
+  when: ansible_system != 'Linux' and not (ansible_facts['system'] == 'FreeBSD' and ansible_facts['distribution_major_version'] is version('14.0', operator='ge'))
+
+- name: install psutil not using pip (FreeBSD >=14.0)
+  command: pkg install py9-psutil
+  when: ansible_facts['system'] == 'FreeBSD' and ansible_facts['distribution_major_version'] is version('14.0', operator='ge')
```

Since the comparison is using `distribution_major_version`, the `version` test isn't correct. This works:

```suggestion
  when: ansible_system != 'Linux' and not (ansible_facts.system == 'FreeBSD' and (ansible_facts.distribution_major_version | int) >= 14)

- name: install psutil not using pip (FreeBSD >=14.0)
  command: pkg install py9-psutil
  when: ansible_facts.system == 'FreeBSD' and (ansible_facts.distribution_major_version | int) >= 14
```